flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BaseWriter.java

```diff
@@ -64,7 +66,6 @@
 abstract class BaseWriter<IN> implements SinkWriter<IN> {
 
     protected final Logger logger = LoggerFactory.getLogger(getClass());
-
     // Multiply 0.95 to keep a buffer from exceeding payload limits.
     private static final long MAX_APPEND_REQUEST_BYTES =
```

Comment:
initalizeMetrics() abstract in base class and then specific implementation in their specific classes.